packages/react/src/__tests__/forwardRef-test.js

```diff
@@ -190,15 +207,16 @@ describe('forwardRef', () => {
         <RefForwardingComponent />
       </p>,
     );
-    await expect(async () => {
-      await waitForAll([]);
-    }).toErrorDev(
+    await waitForAll([]);
+    assertConsoleErrorDev([
       'Each child in a list should have a unique "key" prop.' +
         '\n\nCheck the top-level render call using <ForwardRef>. It was passed a child from ForwardRef. ' +
         'See https://react.dev/link/warning-keys for more information.\n' +
         '    in span (at **)\n' +
-        '    in ',
-    );
+        (gate(flags => flags.enableOwnerStacks)
+          ? '    in **/forwardRef-test.js:**:** (at **)'
```

Comment:
@sebmarkbage not sure if this is expected, but it's possible for owner stacks to have a file path in the `name` like here. I updated the `normalizeCodeLocInfo` to replace it if needed.